Tools/TestWebKitAPI/Tests/WGSL/ParserTests.cpp

```diff
@@ -91,6 +88,21 @@ TEST(WGSLParserTests, Struct)
     EXPECT_EQ(memberType.name(), "i32"_s);
 }
 
+TEST(WGSLParserTests, Struct)
+{
+    // without trailing comma
+    testStruct(
+        "struct B {\n"
+        "    a: i32\n"
+        "}"_s);
+
+    // with trailing comma
+    testStruct(
```

Comment:
What about a test for a struct with two fields?

Reply:
that sounds reasonable, I'll add that. I was just "economical" and reused the existing test